src/Tribe/Admin/Event_Meta_Box.php

```diff
@@ -202,7 +202,7 @@ public function is_auto_draft() {
 		// Fetch Status to check what we need to do
 		$status = get_post_status( $this->event->ID );
 
-		if ( ! is_string( $status ) || 'auto-draft' !== $status ) {
+		if ( ! is_string( $status ) || 'auto-draft' === $status ) {
```

Comment:
Now we're returning true here, is there still value in the `is_string()` test - especially as we're using `===` strict comparison?